users/src/test/java/io/crate/operation/auth/HostBasedAuthenticationTest.java

```diff
@@ -49,17 +50,20 @@
         .put("user", "crate")
         .put("address", "127.0.0.1")
         .put("method", "trust")
+        .put("protocol", "pg | http")
```

Comment:
I think you misunderstood something wrong from the planning doc. the `|` should mean either `pg` OR `http`